Source/santad/DataLayer/WatchItems.h

```diff
@@ -141,7 +166,8 @@ class WatchItems : public std::enable_shared_from_this<WatchItems> {
   NSDictionary *ReadConfig();
   NSDictionary *ReadConfigLocked() ABSL_SHARED_LOCKS_REQUIRED(lock_);
   void ReloadConfig(NSDictionary *new_config);
-  void UpdateCurrentState(DataWatchItems new_data_watch_items, NSDictionary *new_config);
+  void UpdateCurrentState(DataWatchItems new_data_watch_items,
+                          ProcessWatchItems new_proc_watch_items, NSDictionary *new_config);
```

Comment:
Can the last arg be on its own line too?

Reply:
no, formatter changes it back :|

Reply:
😒